.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+name: Alfresco REST API Explorer CI
+
+on:
+  pull_request:
+    branches:
+      - master
+      - fix/**
+      - feature/**
+  push:
+    branches:
+      - master
+      - fix/**
+      - feature/**
+  workflow_dispatch:
+  schedule:
+    - cron: '0 5 * * 3'
+
+env:
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  RELEASE_VERSION: "7.3.0"
+  DEVELOPMENT_VERSION: "7.4.0-A1-SNAPSHOT"
+  MAVEN_CLI_OPTS: "-B -q -e -fae -V -DinstallAtEnd=true -U"
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 30
+
+jobs:
+  pre_commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@v1.34.2
+
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    needs: [pre_commit]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.34.2
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.34.2
+      - name: "Verify"
```

Comment:
It would be nice to keep the "Build" step just before this one, just properly named, as we had it before the last review. This will help with caching the Maven artifacts immediately + stopping the build if the install fails for whatever reason, saving us from running more steps unnecessarily.

Reply:
Done.